### PR TITLE
Version 0.7.0

### PR DESCRIPTION
**Pull request by alexanderdean**

### CODE REVIEW

**Comment by alexanderdean** [comment]
@mhadam @dilyand why is the command here called `-dropOutput`? To me drop means something like delete or remove, but we seem to be doing the opposite - showing it...

**Comment by mhadam** [comment]
@alexanderdean Makes sense, `-showOutput` would be better?

**Comment by alexanderdean** [comment]
How about `-showQueryOutput` to make clear it's the output from the query, not some output from SQL Runner itself? @dilyand?

**Comment by dilyand** [comment]
`-showQueryOutput` looks like a good, descriptive name for it to me.

**Comment by mhadam** [comment]
Ok - all flag changes are made, will update the blog and documentation to reflect it, and everything should be in place for release.

**Comment by alexanderdean** [comment]
Thanks @mhadam - please @ me in the blog post thread when it's ready for me to re-review.

**Comment by alexanderdean** [comment]
Added @BenFradet for code review.

File: sql_runner/scanner.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
+package main
```
Comment (BenFradet) [inline sql_runner/scanner.go]: header

File: sql_runner/scanner.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+		return dst
+	}
+
+	for _, s := range results.results {
+		dst = types.AppendString(dst, s, 1)
+		dst = append(dst, ',')
+	}
+	dst = dst[:len(dst)-1]
+	return dst
+}*/
```
Comment (BenFradet) [inline sql_runner/scanner.go]: ?

File: sql_runner/snowflake_target.go [hunk 3]
```diff
@@ -49,31 +66,95 @@ func (sft SnowFlakeTarget) GetTarget() Target {
...
-				aff, _ := res.RowsAffected()
-				affected += aff
+				res, err := sft.Client.Exec(script)
+
+				if err != nil {
+					return QueryStatus{query, query.Path, int(affected), err}
+				} else {
+					aff, _ := res.RowsAffected()
+					affected += aff
+				}
```
Comment (BenFradet) [inline sql_runner/snowflake_target.go]: this seems very similar to pg, can't this be abstracted

File: vendor/github.com/aws/aws-sdk-go/aws/client/client.go [hunk 4]
```diff
@@ -15,6 +15,12 @@ type Config struct {
 	Endpoint      string
```
Comment (BenFradet) [inline vendor/github.com/aws/aws-sdk-go/aws/client/client.go]: can't we remove those deps from the repo? I know we don't have them in df-runner. Imo they just pollute the repo.

File: sql_runner/snowflake_target.go (hunk 3 above)
Comment (mhadam) [reply sql_runner/snowflake_target.go]: Good point. I'll try to refactor before release.

File: vendor/github.com/aws/aws-sdk-go/aws/client/client.go (hunk 4 above)
Comment (mhadam) [reply vendor/github.com/aws/aws-sdk-go/aws/client/client.go]: they're used for template functions, but if they're not needed I'll remove them

**Pull request merged**